learn/installing-ballerina.md

```diff
@@ -17,44 +17,29 @@ redirect_from:
 
 # Installing Ballerina
 
-- [Updating Ballerina](#updating-ballerina)
 - [Installing Ballerina via installers](#installing-ballerina-via-installers)
 - [Installing via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file)
+- [Updating Ballerina](#updating-ballerina)
 - [Building from source](#building-from-source)
 - [Uninstalling Ballerina](#uninstalling-ballerina)
 - [Getting help](#getting-help)
 - [What is next](#what-is-next)
 
-## Updating Ballerina
-
-If you already have a jBallerina version installed, you can use the update tool (by executing the below commands) to update to the latest jBallerina version as follows. 
-
->**Note:** If you are a new user who has not installed jBallerina, install via the [installers](#installing-ballerina-via-installers).
-
-**The jBallerina Version**|**Update Tool Usage**
-:-----:|:-----:
-jBallerina version 1.2.0 or above|`ballerina dist update`
-jBallerina 1.2.0 but switched to a previous version|`ballerina dist pull jballerina-<LATEST-VERSION>`
-jBallerina version 1.1.0 or below|Use the [installers](#installing-ballerina-via-installers)
-
-
 ## Installing Ballerina via installers
 
-Ballerina installers support the below operating systems:
+Follow the below instructions to download and use the installers to install the latest Ballerina version. The installer will automatically uninstall the old Ballerina version if you have one already installed.
```

Comment:
Shall we edit the first line to this: 

Follow the instructions below to install the latest Ballerina version using the installer.

The sentence here seems like a mouthful. WDYT?

Reply:
Fixed.